configs/yoloxpai/yolox_pai_asff_s_8x8b-300e_coco.py

```diff
@@ -0,0 +1,3 @@
+_base_ = './yolox_pai_s_8x8b-300e_coco.py'
+
+model = dict(neck=dict(type='YOLOXASFFPAFPN'))
```

Comment:
ASFF should be used in series with PAFPN to facilitate subsequent reuse.

neck=[dict(),dic(type=ASFF)]

Reply:
yolox pafpn: top-down path -> bottom-up path -> out convs
yolox asff pafpn: top-down path -> bottom-up path -> asff -> out convs

So, is `yolox pafp neck(wo out convs) -> asff neck(w out convs)` okay?

Reply:
This seems to have bc breaking, so let's go with the current practice.

Reply:
https://github.com/alibaba/EasyCV/blob/4fa81beb217a8b0ee2108ac352bf67196a52bcba/easycv/models/detection/detectors/yolox/yolo_head_template.py#L87

Here it is.